wikimedia/wikipedia-ios
# Onboarding data policy update

**Phabricator:** https://phabricator.wikimedia.org/T356206

### Notes
* This PR updates our data policy not to allow opt-out of collecting anonymous data
* The Welcome Onboarding VC was updated to remove the toggle and show the new copy
* The data-sharing verifications throughout the code were removed
* The session ID generation was updated since it's not attached to a user action

### Test Steps
1. Run a fresh install of the app, go through onboarding
2. On the data & privacy screen, verify copy, layout, and links
3. Verify that the session ID is being created and reset correctly
